pkg/jobrunaggregator/releasebigqueryloader/release_uploader.go

```diff
@@ -206,6 +207,23 @@ func releaseDetailsToBigQuery(architecture string, tag ReleaseTag, details Relea
 		release.Release = strings.Join(parts[:2], ".")
 	}
 
+	// Get "nightly" or "ci" from the string
```

Comment:
I don't know when and I don't know how, but this will break someday.  Try to remember it's optional in your consuming code.